src/app/login-signup/login/login.component.ts

```diff
@@ -1,10 +1,50 @@
-import { Component } from '@angular/core';
 
+import { Component ,OnInit} from '@angular/core';
+import { FormBuilder, FormGroup, Validators , FormControl} from '@angular/forms';
+import { ActivatedRoute,Router } from '@angular/router';
+import { ReactiveFormsModule } from '@angular/forms';
+
+// an interface that contains the properties of username and password.
+interface User {
```

Comment:
create model for this. do not change right now but while implementing it, please remove interface and create modal

Reply:
I will do changes in next PR